Navigation Update

emdecr opened the pull request:
#### Summary
Hi Team, this PR syncs the navigation with what is seen on mm.com:

- Main nav/menu gets an addition of a secondary CTA button and updated CTA links
- Footer changes in the legal/disclaimer (style update + addition of Cookie Policy) and social (add LinkedIn) sections.

TY + let me know if you have any questions!

cc: @cwarnermm

cwarnermm commented:
@emdecr - Potentially related to this scope of this PR: The footer of https://mattermost.com/privacy-policy/cookies/ contains different links than the docs.mm.com footer (A "Terms" link is missing from docs). However, the "Terms" link appears to take users to https://mattermost.com/support/. Is that intended?

emdecr commented:
> @emdecr - Potentially related to this scope of this PR: The footer of other web assets, such as https://mattermost.com/privacy-policy/cookies/, contains a different set of links than the docs.mm.com footer (A "Terms" link is missing from docs). However, the "Terms" link appears to take users to https://mattermost.com/support/. Is that intended?

That is a really good point. I did miss syncing a few of the footer links. With my latest push, I've adjusted some wording under the Documentation section, and added a Terms link under Support - and yep, it is supposed to go to mm.com/support because the page lists the Terms of our Support services. 

Thanks so much for catching this + let me know if you have any questions~!

cwarnermm reviewed (approved):
The latest updates look great @emdecr! Thank you!
